.github/workflows/ci.yml

```diff
@@ -63,13 +63,15 @@ jobs:
       - uses: actions/setup-node@v4
         with:
           node-version-file: .nvmrc
-          cache: yarn
+          cache: npm
 
-      - run: yarn install
+      - run: npm install
 
-      - run: yarn playwright install
+      - run: npx playwright install
 
-      - run: yarn test:e2e
+      - run: |
+          npm run build
```

Comment:
@maxammann Are you sure this line is needed? I thought that was for Vercel, which isn't used for end-to-end testing at this time.